### PR TITLE
p2-replicate: Use text logger

### PR DESCRIPTION
**Pull request by petertseng**

### CODE REVIEW

File: bin/p2-replicate/main.go [hunk 1]
```diff
@@ -68,6 +69,7 @@ func main() {
 	allocated := allocation.NewAllocation(*hosts...)
 
 	replicator := replication.NewReplicator(*manifest, allocated)
+	replicator.Logger.Logger.Formatter = new(logrus.TextFormatter)
```
Comment (petertseng) [inline bin/p2-replicate/main.go]: logger logger

Comment (anthonybishopric) [inline bin/p2-replicate/main.go]: Yeah, a little weird. Not sure it's worth doing anything about

**Comment by petertseng** [comment]
Tested in universe, looking nice and colorful.

**Comment by anthonybishopric** [comment]
:+1:

**Pull request merged**